blocks/header/__link/link.css

```diff
@@ -0,0 +1,10 @@
+.link {

```

Comment:
These styles seem to be misplaced and should be moved to `link` mixin file which is currently empty. 

If you will no longer need of header__link element, please consider removing it to keep the code clean and organized. Don't forget HTML and page.css import 😊

Reply:
Done🙂